### ISSUE TITLE
can no longer switch to another workspace

### ISSUE THREAD

**Issue by netfab** [ValveSoftware/steam-for-linux#10545]
#### Your system information

* Steam client version (build number or date): 1708985249
* Distribution (e.g. Ubuntu): Gentoo
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* Steam Logs: [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/14430553/steam-logs.tar.gz)
* GPU: Nvidia/Intel

#### Please describe your issue in as much detail as possible:

Since last table steam update, when steam is running, I can no longer switch to another workspace. Steam brings me back automagically to the workspace where it is. Have a look to the following video. 
With the mouse pointer at the bottom of the screen, I'm trying to switch to other workspaces, but Steam brings me back : https://github.com/ValveSoftware/steam-for-linux/assets/1150555/5e7cbbf9-c5c1-42e4-aef1-bdefaa84ca4b

I'm under Gentoo - Xfce-4.18 - Xfwm 4.18.0.

#### Steps for reproducing this issue:

1. run steam
2. in the desktop environment, try to move to another workspace

**Comment by netfab** [comment]
This behavior disappears after running/exiting a game. Then, when steam is restarted, this come back.

**Comment by zelurker** [comment]
Yeah they broke most window managers in their latest update, and they don't seem to even read their issues since this project has the amazing number of 2.6k issues... !!! With such a level of contempt for their users, it's incredible to think that the steam deck is supposed to work in linux...
For me in windowmaker I can move to another workspace but the steam window is not updated and when I come back, it's all black. More annoying : each time the steam window loses focus, it's moved down a few pixels, if you have some flying focus it makes things unmanageable if there is more than 1 window on screen (just the steam window, alone).
And that's years after creating the linux client, there is a thread asking for more compatibility with window managers which has an incredible number of posts and which is totally ignored by them, there is also an early beta open steam client attempt by the community, but for now it's not really usable. Anyway it's a total mess !
